tools/test

```diff
@@ -7,7 +7,7 @@ SLOW=
 TAGS=
 PARALLEL=
 RACE="-race -covermode=atomic"
-TIMEOUT=5m
+TIMEOUT=10m
```

Comment:
Tests run with `-race` in CI and, when race is enabled, they're very slow, especially new "stress" tests I've added for shuffle sharding. However, at this stage I believe we should prefer coverage over speed. We'll later work to speed up CI.